maker/src/trading/bitmex_client.rs

```diff
@@ -34,9 +36,8 @@ pub async fn bitmex(network: Network) -> impl Stream<Item = Result<Quote, Error>
                 }
             }
 
-            let seconds = 10;
-            tracing::warn!("Disconnected from BitMEX. Reconnecting to BitMEX in {seconds}");
-            tokio::time::sleep(Duration::from_secs(seconds)).await;
+            tracing::warn!("Disconnected from BitMEX. Reconnecting to BitMEX in {seconds} seconds", seconds = BITMEX_RECONNECT_INTERVAL.as_secs());
```

Comment:
🔧Shouldn't this be like this?

```rust
tracing::warn!("Disconnected from BitMEX. Reconnecting to BitMEX in {} seconds", BITMEX_RECONNECT_INTERVAL.as_secs());
```

I don't know what the current code does, because it seems to be combining string interpolation with `tracing` attributes.

Reply:
I don't mind changing.